src/main/scala/org/globalforestwatch/summarystats/annualupdate_minimal/AnnualUpdateMinimalDF.scala

```diff
@@ -105,7 +105,7 @@ object AnnualUpdateMinimalDF {
         $"data_group.aze" as "is__birdlife_alliance_for_zero_extinction_site",
         $"data_group.plantedForests" as "gfw_plantation__type",
         $"data_group.mangroves1996" as "is__gmw_mangroves_1996",
-        $"data_group.mangroves2016" as "is__gmw_mangroves_2016",
+        $"data_group.mangroves" as "is__gmw_mangroves_latest",
```

Comment:
Questions on `isGain` and `totalGainArea` below-- why is is 2000-2012 now, not 2000-2020? Is there another 2000-2020 one?

Reply:
Spoke to @jterry64, we're keeping `2000-2012` in the column name for now to not break Flagship queries. The layer correctly points to `umd_tree_cover_gain_from_height/v202206`.

Reply:
Okay. That's definitely confusing. I hope you can change that in the next version.